dashboard/test/ui/features/teacher_tools/script_overview.feature

```diff
@@ -62,21 +62,13 @@ Feature: Unit overview page
   Scenario: Unit overview end-of-lesson
     Given I create a student named "Jean"
     # On last level of the lesson
-    And I am on "http://studio.code.org/courses/csp-2019/units/3/lessons/3/levels/1"
+    And I am on "http://studio.code.org/courses/csp-2025/units/1/lessons/7/levels/8"
     And I click selector ".submitButton"
-    And I wait until I am on "http://studio.code.org/courses/csp-2019/units/3"
+    And I wait until I am on "http://studio.code.org/courses/csp-2025/units/1"
     And I wait for jquery to load
-    And I wait until element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is visible
+    And I wait until element ".uitest-end-of-lesson-header:contains(You finished Lesson 7!)" is visible
     And I reload the page
-    And  element ".uitest-end-of-lesson-header:contains(You finished Lesson 3!)" is not visible
-
-  Scenario: Unit overview lesson plan
```

Comment:
Lesson plans are now at a different url that is tested in `Unit overview new lesson plan`.